### ISSUE TITLE
UI automation in Word doesn't report paragraph indentation

### ISSUE THREAD

**Issue by vincent-lg** [nvaccess/nvda#10192]
When turning on UI automation for Word in the advanced settings, NVDA+F fails to report the paragraph indentation.

### Steps to reproduce

1. Turn on UI automation in Word.
2. Open a new blank document.
3. Type some text.
4. Go to the beginning of the paragraph and press the tab key.  This increases indentation by something like half an inch for the current line.  This increase is correctly reported.
5. Press NVDA+F.

### Output

If UI automation isn't enabled, the paragraph indentation (0.5 inch for me) is reported.  This information is missing, however, with UI automation turned on.

### More info

- NVDA (2018.2 or `alpha-18590,50619158` have this problem).

Thanks,

**Comment by Adriani90** [comment]
cc: @michaelDCurran
